music_assistant/providers/audible/__init__.py

```diff
@@ -270,14 +283,18 @@ async def get_library_audiobooks(self) -> AsyncGenerator[Audiobook, None]:
 
     async def get_audiobook(self, prov_audiobook_id: str) -> Audiobook:
         """Get full audiobook details by id."""
-        audiobook = await self.helper.get_audiobook(asin=prov_audiobook_id, use_cache=False)
-        if audiobook is None:
-            raise ValueError(f"Audiobook with id {prov_audiobook_id} not found")
-        return audiobook
+        try:
+            return await self.helper.get_audiobook(asin=prov_audiobook_id, use_cache=False)
+        except MediaNotFoundError as err:
+            raise MediaNotFoundError(f"Audiobook with id {prov_audiobook_id} not found") from err
```

Comment:
```suggestion
        return await self.helper.get_audiobook(asin=prov_audiobook_id, use_cache=False)
```

Reply:
the try..except was redundant here because it was already raising MediaNotFoundError